MoyaLogger.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+
+
+public struct MoyaLogger: Logger {
+    
+    private let loggerId = "Moya_Logger"
+    private let dateFormatString = "dd/MM/yyyy HH:mm:ss"
+    private let dateFormatter = NSDateFormatter()
+    
+    private var date: String {
+        
+        dateFormatter.dateFormat = dateFormatString
+        dateFormatter.locale = NSLocale(localeIdentifier: "en_US_POSIX")
+        return dateFormatter.stringFromDate(NSDate())
+    }
+    
+    public func logNetworkRequest(request: NSURLRequest) {
+        
+        let requestOutput = String(format: "%@: [%@] Request:  %@", loggerId, date, request.description)
```

Comment:
Yeah, well if we aggregate the log output anyway, as discussed, repeating the preamble in the same string seems superfluous. Again, up to you!